### ISSUE TITLE
[Bug]: Firefox: TypeError: 'camera' (value of 'name' member of PermissionDescriptor) is not a valid value for enumeration PermissionName

### ISSUE THREAD

**Issue by RUSSCITY** [Baseflow/flutter-permission-handler#1308]
### Please check the following before submitting a new issue.

- [X] I have searched the [existing issues](https://github.com/baseflow/flutter-permission-handler/issues).
- [X] I have carefully [read the documentation](https://github.com/Baseflow/flutter-permission-handler/blob/main/permission_handler/README.md) and verified I have added the required platform specific configuration.

### Please select affected platform(s)

- [ ] Android
- [ ] iOS
- [X] Windows

### Steps to reproduce

Request `await Permission.camera.status` when running the app on web in Firefox, or
request: `await Permission.camera.isGranted`

### Expected results

I expect to get the current permission status

### Actual results

I get this error:
```
TypeError: 'camera' (value of 'name' member of PermissionDescriptor) is not a valid value for enumeration PermissionName
```

### Code sample

<details><summary>Code sample</summary>

```dart
      final status = await Permission.camera.status;
```

</details>

### Screenshots or video

<details>
<summary>Screenshots or video demonstration</summary>

[Upload media here]

</details>

### Version

11.3.1

### Flutter Doctor output

<details><summary>Doctor output</summary>

```console
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.19.5, on Microsoft Windows [Version 10.0.22631.3447], locale de-DE)
[√] Windows Version (Installed version of Windows is version 10 or higher)
[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[√] Chrome - develop for the web
[√] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.9.5)
[√] Android Studio (version 2023.2)
[√] Connected device (3 available)
[√] Network resources

```

</details>

**Comment by Arthurius** [comment]
I have the same issue. It also happens with `Permission.microphone`

**Comment by reza00farjam** [comment]
<Commenting to keep the thread open>

**Comment by mvanbeusekom** [comment]
@Arthurius, this issue has been solved 14 days ago in version 0.1.3+2 of the permission_handler_html package. 

Please try to update to the latest version by running `flutter pub upgrade permission_handler_html` and let us know if you still run into issues.
